examples/typescript-enum-type/index.spec.ts

```diff
@@ -7,17 +7,10 @@ describe('Should be able to render correct enums based on options', () => {
   afterAll(() => {
     jest.restoreAllMocks();
   });
-  test('and should log expected output to console', async function testIt(): Promise<void> {
-    // Wait for first call to the `generate` function is over, and we can reliably know the output
-    if (spy.mock.calls.length !== 6) {
-      await timeout(50);
-      return testIt();
-    }
-
+  test('and should log expected output to console', async (): Promise<void> => {
     await generate();
-    //Generate is called 2 times, so even though we expect 6 console logs we double it
-    expect(spy.mock.calls.length).toEqual(12);
-    const generatedContent = spy.mock.calls.slice(6).join('\n');
+    expect(spy.mock.calls.length).toEqual(6);
+    const generatedContent = spy.mock.calls.slice(3).join('\n');
```

Comment:
```suggestion
    const generatedContent = spy.mock.calls.join('\n');
```

Reply:
now its working fine and all testcases passed. Just one more doubt, should i commit latest snapshots ?

Reply:
Yes please :)

Reply:
done :heavy_check_mark: